.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+**Fixes issue:  #[]**
```

Comment:
What I like, and is my personal preference only, is having these things in a PR
What is the problem i'm trying to address 
How is fix applied (short description of the solution)

Fixes #xxx
Fixes #yyy

those fixes at the end, prefexied by Fixes keyword

Reply:
I agree, it's mostly preference. I like using `closes` but either way if we use something like `fixes #324` GitHub will close the issue when the PR is merged. You can even have a comma separated list of issues and or pull requests that it will close when merged.